### DISCUSSION TITLE
BirdNET-Pi, BirdWeather Privacy

### DISCUSSION THREAD

**Discussion by mikekeith** [mcguirepr89/BirdNET-Pi#1058] [Q&A]
hi there,

i finally got around to setting up a BirdNET-Pi a few days ago and absolutely love it :)

i've got a Birdweather ID and was wondering, is there a way to toggle on/off the sending of data to Birdweather ? The microphone is currently placed close to the house, so i've been going into the birdnet-pi settings and setting/unsetting the ID manually depending on if we're active near the station etc.

i haven't dug into it, but if there isn't a webfrontend endpoint but i'm guessing i could script modifying the config and restart the service? has anyone tried that before?

also, i'm unsure about what data is sent to the birdweather site.. is it the audio recordings saved in birdnet-pi when it detects a bird? (i.e. occasional snippets of audio where birds were detected), or a stream?

**Comment by waynecarnold** [comment]
On the webpage for the birdnetPi settings you can set the option to ignore human voices. I have mine set and any time a human sound is detected (sometimes even things like chair moving etc) it is ignored and not uploaded). I cannot send more details as my Pi is at my cottage so I cannot get to the web interface to manage settings. I believe if I remember correctly all files are stored in the local database and only when a recording meets the criteria you configured in settings, will it be uploaded. You can watch the log file and you will see this in real time. Hope that helps.

**Reply by mikekeith** [reply]
thanks! much appreciated - that's good too know :) i realised some of the other settings needed tweaking too, so hopefully those + privacy threshold will do the trick.

**Comment by waynecarnold** [comment]
Also it might be useful to look at this [thread](https://github.com/mcguirepr89/BirdNET-Pi/discussions/1016#discussioncomment-7005587).
